### PR TITLE
Update for Multi CA, Accelerated Networking and runCommands. 

### PR DESCRIPTION
**Pull request by hyonholee**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@hyonholee please add needs-review once you are ready for merge.

File: src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("15.0.0.0")]
-[assembly: AssemblyFileVersion("15.0.0.0")]
+[assembly: AssemblyVersion("16.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs]: @hyonholee just curious, why AssemblyVersion and nuget version differed earlier (nuget version being 15, but AssemblyVersion 16)
And now why update from 16.0 to 16.2? why skip 16.1?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (hyonholee) [inline src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs]: @shahabhijeet   It should be 16.1.   I updated this PR.  I don't know why the AssemblyVersion was not updated last time.  It looks like a mistake last time.

**Review by shahabhijeet — APPROVED** [review]

**Comment by saguiitay** [comment]
Any update on when this will be released? I need to be able to read/set the AcceleratedNetwork property..

**Pull request merged**